functions/consumer/tcp-consumer/src/main/java/org/springframework/cloud/fn/consumer/tcp/TcpConsumerConfiguration.java

```diff
@@ -38,28 +37,28 @@
  *
  * @author Gary Russell
  * @author Christian Tzolov
+ * @author Chris Bono
  */
-@Configuration
+@Configuration(proxyBeanMethods = false)
```

Comment:
This is what I asked long time ago to revise...

Thank you, @onobc , for starting this effort!

Reply:
No worries! Yes, I have tried to do this in every single `@Configuration` I have touched since Boot 2.2 (or whenever it was added).